tests/main_test.go

```diff
@@ -1,165 +0,0 @@
-package tests
```

Comment:
This test is kind of integration or system test. I see you removed it. We can keep these test and just use gRPC for querying the node.

Reply:
> @braewoods Thanks. All looks good. JUst the `tests` folder is removed. That test is really important to make sure the system works fine

I thought they were exclusive to CAPNP due to how tightly integrated to it they seemed to be which is why they were removed.